tests/handlers/external_test.py

```diff
@@ -91,3 +96,89 @@ async def test_current_channels(
                 else:
                     assert parsed.select(".event-info")
                     assert not parsed.select(".event-error")
+
+
+@pytest.mark.asyncio
+async def test_all_endpoints(
+    mocked_client: tuple[AsyncClient, FastAPI, RubinDataMocker]
+) -> None:
+
+    client, app, mocker = mocked_client
+
+    locations = [loc.name for loc in m.locations]
+    summit: Location = find_first(m.locations, "name", "summit-usdf")
+    assert isinstance(summit, Location)
+
+    all_cams = [f"{summit.name}/{cam.name}" for cam in summit.cameras]
+    online_cams = [f"{summit.name}/{cam.name}" for cam in summit.cameras if cam.online]
+    camera_historical = [f"{cam}/historical" for cam in online_cams]
+    camera_dates = [f"{cam}/date/{day_obs}" for cam in online_cams if cam]
```

Comment:
Why does only this case use the `if cam` checking? (when iterating over `online_cams`).

Reply:
Good spot, thanks! It's redundant.